BottomBar.Droid/Renderers/BottomBarPageRenderer.cs

```diff
@@ -227,7 +231,8 @@ void UpdateBarTextColor ()
 				return;
 			}
 
-			// haven't found yet how to set text color for tab items on_bottomBar, doesn't seem to have a direct way
+            _bottomBar.SetActiveTabColor(Element.BarTextColor.ToAndroid());

```

Comment:
And please may I ask for a bit more friendly reactions? Because as in my case I was not aware of any formating problems.